Waste system organ tweaks

comma opened the pull request:
Liver: no more straight up damage at bad filtering levels, creates ammonia up to toxic levels instead.
Can fight it with dialysis now.

Kidneys: added potassium generation when damaged.

Potassium increases pulse, dangerously so in amounts over 10 units.
Added some potassium to banans for monkey chemistry.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

PurpleMartinJCK commented:
Why specifically ammonia and potassium?

comma commented:
lolrealism pretty much, and ammonia would be toxic without having buckets of it in bloodstream

Ratvar commented:
Have damaged kidneys
Drink cup of water
Pop

comma commented:
pop pop
(actually not, stomach and bloodstream are separate holders, regents do not travel between)

PurpleMartinJCK reviewed (approved):
Skeptical but let's see how it goes.

comma committed:
Waste system organ tweaks

Liver: no more straight up damage at bad filtering levels, creates ammonia up to toxic levels instead.
Can fight it with dialysis now.

Kidneys: added potassium generation when damaged.

Potassium increases pulse, dangerously so in amounts over 10 units.
Added some potassium to banans for monkey chemistry.
